packages/react-chat/src/components/Image/Default.tsx

```diff
@@ -6,17 +6,24 @@ import { VariantProp } from '@/types';
 export const tag = tagFactory(ClassName.IMAGE);
 
 export const DefaultImageBase = styled(tag('img'), {
-  width: 248,
-
   variants: {
     isRounded: {
       true: {
         borderRadius: '$1',
       },
     },
+    mode: {
+      bubble: {
+        width: 248,
+      },
+      embedded: {
+        width: '100%',
+      },
+    },
```

Comment:
In my testing idk if we want to do this, this leads to images being way too big when stretched out horizontally. There's nowhere else in our app where a img can get that big.

If you have a very wide chat and a vertical image, it gets absolutely too gigantic.

I actually might prefer it to being static 248px.

Reply:
the last time we were testing it with Daniel we agreed on stretching, because if the image is wide but short, it will be really hard to read if it's not stretched to 100%

let's clarify